Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -1,18 +1,36 @@
 # Preface
 This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they cannot be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for your incident.
 
-## When it is OK to engage the Triage Team
+## Is this is a Severity 1 Issue? - Notify On Call Immediatley
 
-* When the incident is believed to be **Severity 1** which can be defined as
-  * A significant amount of users are impacted
-  * Issue resolution is required in a 24 hour period
-  * Part of (or all of) va.gov is severely disrupted
+When the incident is believed to be **Severity 1** which can be defined as (but not limited to)
+ * Issue resolution is required in a 24 hour period
+ * Part of (or all of) va.gov is severely disrupted
   
-* When an issue is discovered that is not in your products scope of work
-  * Any issue that is discovered that is outside your teams scope of work can be reported to Triage
-  * In this case severity does not matter
+Examples:
+- va.gov is unresponsive
+- Platform errors causing extreme delays for users
+  
+If you believe the issue falls into the Severity 1 category, there is no need to follow the Triage process.  Immediate notification to On Call is the best path to resolution.
+
+Slack - [#oncall](https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Practice%20Areas/Engineering/OnCall)
+
+[Instructions on escalting to On Call can be found here](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/Incident%20Response%20Playbook.md)
+
+## When to engage the Triage Team
+  
+**General Rule:** Any issue that is discovered that is outside your teams scope of work can be reported to Triage
```

Comment:
We could provide examples here as well; similar to the examples for a sev 1 issue. We already have some real world examples 🙂 :

```
Examples:
- An internal load testing tool is broken
- Mock data is not working or is out of date
- Metrics are being reported incorrectly or not reported at all
```